samples/tfm_psa_test/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
We should update the version requirement for CMake.
Some generator expressions did not work with CMake 3.13.3, but worked with CMake 3.18.4.

Reply:
Do you remember which generator expressions?

Reply:
and we should only update to when those expressions were new, or even better, re-write them to `3.13.` versions if possible.